### ISSUE TITLE
Open data sets we can use to practice

### ISSUE THREAD

**Issue by char-siuu-bao** [learningd3together/learningspace#5]
I think the best way to learn is by doing + practicing on real data sets that we are motivated to explore and curious about. If we come across an interesting-looking open data set that you want to share, drop the link down here!  (simple is probably better for those of us who are beginners to d3 - me included!)

**Comment by ghost** [comment]
I totally agree. I'm a beginner in d3.js too. It's my first time with the framework.

**Comment by weskerfoot** [comment]
https://data.cdc.gov/ might be good

**Comment by char-siuu-bao** [comment]
There's also a bunch from the Pew Research Centre! http://www.pewsocialtrends.org/datasets/
